src/Resources/NginxTemplate.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Laravel\Forge\Resources;
+
+class NginxTemplate extends Resource
+{
+    /**
+     * The id of the nginx template.
+     *
+     * @var int
+     */
+    public $id;
+
+    /**
+     * The id of the server.
+     *
+     * @var int
+     */
+    public $serverId;
+
+    /**
+     * The name of the nginx template.
+     *
+     * @var string
+     */
+    public $name;
```

Comment:
Don't we also need a property for the template's content?

Reply:
Done in this commit https://github.com/laravel/forge-sdk/pull/107/commits/ca48f5d649e5972c4e264b3a4be1bb67829060a5.